spacelift/resource_idp_group_mapping.go

```diff
@@ -37,20 +39,20 @@ func resourceIdpGroupMapping() *schema.Resource {
 		Schema: map[string]*schema.Schema{
 			"name": {
 				Type:             schema.TypeString,
-				Description:      "Name of the user group - should be unique in one account",
+				Description:      "Name of the IdP group as defined in the SSO provider - should be unique per account",
 				Required:         true,
 				ForceNew:         true,
 				ValidateDiagFunc: validations.DisallowEmptyString,
 			},
 			"policy": {
-				Type:     schema.TypeSet,
-				MinItems: 1,
-				Required: true,
+				Type:        schema.TypeSet,
+				Description: "(Legacy) List of access rules for the IdP group. Prefer using the `spacelift_role_attachment` resource instead for permission management.",
```

Comment:
@KNiepok @mwasilew2 @michieldewilde @ilya-hontarau 

Are you guys okay with me marking "user group attachments" legacy here, or do we want to with with that until the full release?

Reply:
I would not worry about marking it as legacy until we have released the GA, after that we can do the following: https://developer.hashicorp.com/terraform/plugin/framework/deprecations#provider-attribute-removal

Reply:
okay, I'll remove the legacy mentions for now